mixer/adapter/bypass/config/config.proto

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// $title: Bypass
+// $description: Adapter for connecting to gRPC based remote adapters via the inline adapter model.
+// $location: https://istio.io/docs/reference/config/policy-and-telemetry/adapters/bypass.html
+// $front_matter: redirect_from: /docs/reference/config/adapters/bypass.html
+
+// The `bypass` adapter makes it possible to connect to gRPC based remote adapters using the inline adapter
+// model.
+//
+// This adapter supports the [metric template](https://istio.io/docs/reference/config/policy-and-telemetry/templates/metric.html).
+package adapter.bypass.config;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "config";
+option (gogoproto.goproto_getters_all) = false;
+option (gogoproto.equal_all) = false;
+option (gogoproto.gostring_all) = false;
+
+// Configuration format for the Bypass adapter.
+message Params {
+    // The backend address for the remote policy backend.
+    string backend_address = 1;
+
+    // The actual config parameters to send to the remote backend.
+    google.protobuf.Any params = 2;
```

Comment:
This config is coming from CR and I am not sure how users will be able to write Any type. Do you think we put another message here that the adapter code can transform into Any to pass to the backend. WDYT ?

Reply:
Can we show example YAML for this?